debug/7053_geolocation_follow.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Does this file really need to be committed? It doesn't seem to do anything other than add the geolocate control.